### PR TITLE
Get the output mg-helper pod in failure

### PR DESCRIPTION
**Pull request by OdedViner**
Signed-off-by: Oded Viner <oviner@redhat.com>

### CODE REVIEW

File: ocs_ci/ocs/utils.py [hunk 1]
```diff
@@ -915,6 +917,28 @@ def run_must_gather(log_dir_path, image, command=None):
...
 
+def get_helper_pods_output():
+    """
+    Get the output of "oc describe mg-helper pods"
+
+    Returns:
+        output_describe_mg_helper (str): the output of "oc describe mg-helper pods"
+
+    """
+    from ocs_ci.ocs.resources.pod import get_pod_obj
```
Comment (avd-sagare) [inline ocs_ci/ocs/utils.py:928]: Can we move this to top

File: ocs_ci/ocs/utils.py [hunk 2]
```diff
@@ -916,6 +918,32 @@ def run_must_gather(log_dir_path, image, command=None):
     return mg_output
 
 
+def get_helper_pods_output():
+    """
+    Get the output of "oc describe mg-helper pods"
+
+    Returns:
+        output_describe_mg_helper (str): the output of "oc describe mg-helper pods"
```
Comment (ebenahar) [inline ocs_ci/ocs/utils.py:926]:
```
str: the output of "oc describe mg-helper pods"
```

File: ocs_ci/ocs/utils.py [hunk 3]
```diff
@@ -916,6 +918,32 @@ def run_must_gather(log_dir_path, image, command=None):
...
+
+    """
+    from ocs_ci.ocs.resources.pod import get_pod_obj, get_pod_logs
```
Comment (ebenahar) [inline ocs_ci/ocs/utils.py:929]: is this imported here for preventing a circular import?

Comment (OdedViner) [reply ocs_ci/ocs/utils.py:929]: yes

File: ocs_ci/ocs/utils.py (hunk 2 above)
Comment (OdedViner) [reply ocs_ci/ocs/utils.py]: done

**Review by ebenahar — APPROVED** [review]

**Comment by OdedViner** [comment]
> PR validation on existing cluster
> 
> Cluster Name: oviner9-ui Cluster Configuration: PR Test Suite: tier1 PR Test Path: tests/manage/z_cluster/test_must_gather.py Additional Test Params: OCP VERSION: 4.11 OCS VERSION: 4.11 tested against branch: master
> 
> [Job UNSTABLE](https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-trigger-test-pr/3348//testReport) (some or all tests failed).

Failed because `odf-csi-addons-operator.yaml` file does not exist
```
Exception: Files don't exist:
['odf-csi-addons-operator.yaml']
```
fixed here https://github.com/red-hat-storage/ocs-ci/pull/6283

**Review by petr-balogh — APPROVED** [review]

**Pull request merged**